Support go1.19

**Is your feature request related to a problem? Please describe.**

Compiling using go1.19 will failed with the following messages:
../../../../go/pkg/mod/github.com/cloudwego/frugal@v0.1.1/internal/loader/funcdata_invalid.go:27:9: panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18") (no value) used as value

**Describe the solution you'd like**
support compile using go1.19

**Describe alternatives you've considered**

**Additional context**

--- a/internal/loader/funcdata_go118_119.go
+++ b/internal/loader/funcdata_go118_119.go
@@ -1,4 +1,4 @@
-// +build go1.18,!go1.19
+// +build go1.18,!go1.20
 
 /*
  * Copyright 2022 ByteDance Inc.

--- a/internal/loader/funcdata_invalid.go
+++ b/internal/loader/funcdata_invalid.go
@@ -1,4 +1,4 @@
-// +build !go1.15 go1.19
+// +build !go1.15 go1.20
 
 /*
  * Copyright 2022 ByteDance Inc.
@@ -24,9 +24,9 @@ import (
 
 // triggers a compilation error
 const (
-    _ = panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18")
+    _ = panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18, 1.19")
 )
 
 func registerFunction(_ string, _ uintptr, _ uintptr, _ rt.Frame) {
-    panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18")
+    panic("Unsupported Go version. Supported versions are: 1.15, 1.16, 1.17, 1.18, 1.19")
 }

